app/static/lang/vi-VI.js

```diff
@@ -78,6 +78,12 @@ export default () => {
         wallets: 'Ví',
         title: 'Danh  những ví Lotus đang khả dụng trong hệ thống:',
       },
+      banner: {
+        why_1: 'Những cộng đồng gặp khó khăn trong việc cấp vốn cho các dự án công cộng - bằng việc phân bổ nguồn vốn vào những công việc cấp thiết nhưng không có lợi ích kinh tế. Với mục tiêu này, Lotus đá được tạo ra. Chúng tôi tin tưởng rầng bằng cách thay đổi nhìn nhận về đồng tiền, và thách thức những sáng kiến xung quanh việc sử dụng đồng tiền, chúng tôi có thể giải quyết được việc cấp vốn cho những dự án công cộng.',
```

Comment:
Can you change why_1 to this
'Xã hội thường gặp khó khăn trong việc cấp vốn cho các dự án công cộng - việc phân bổ nguồn lực vào những công việc cấp bách nhưng không có lợi ích kinh tế. Với vấn đề này, Lotus đá được tạo ra. Chúng tôi tin tưởng rầng bằng cách thay đổi cách nhìn nhận về đồng tiền, và thách thức những sáng kiến xung quanh việc sử dụng đồng tiền, chúng tôi có thể giải quyết được việc cấp vốn cho những dự án công cộng.'